libr/util/str.c

```diff
@@ -1882,6 +1882,53 @@ R_API bool r_str_char_fullwidth (const char* s, int left) {
 
 }
 
+/**
+ * Returns size in bytes of the utf8 char
+ * Returns 1 in case of ASCII
+ * str - Pointer to buffer
+ */
+R_API int r_str_utf8_charsize(const char *str) {
```

Comment:
int len , or check for null at the end, because i think all those strings are null terminated

Reply:
isn't that what I have below?
`int length = strlen (str);`
`while (size < length)`